### PR TITLE
Added SphereBasedTagalong + test scene

### PR DESCRIPTION
**Pull request by stbertou**
See issue https://github.com/Microsoft/HoloToolkit-Unity/issues/111

### CODE REVIEW

File: Assets/HoloToolkit/Utilities/Scripts/SphereBasedTagalong.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// A Tagalong that stays at a fixed distance from the camera and always
+    /// seeks to stay on the edge or inside a sphere that is straight in front of the camera.
+    /// </summary>
+    public class SphereBasedTagalong : MonoBehaviour
+    {
+        /// <summary>
+        /// Sphere radius
```
Comment (NeerajW) [inline Assets/HoloToolkit/Utilities/Scripts/SphereBasedTagalong.cs]: NIT: Please convert these to ToolTip.
I have heard feedback from folks that tool tips work better than summary comments in editor.

**Comment by NeerajW** [comment]
Please update the README.md

File: Assets/HoloToolkit/Utilities/Scripts/SphereBasedTagalong.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// A Tagalong that stays at a fixed distance from the camera and always
+    /// seeks to stay on the edge or inside a sphere that is straight in front of the camera.
+    /// </summary>
+    public class SphereBasedTagalong : MonoBehaviour
+    {
+        /// [Tooltip("Sphere radius.")]
```
Comment (NeerajW) [inline Assets/HoloToolkit/Utilities/Scripts/SphereBasedTagalong.cs]: Does /// mean they are commented out?
Will this mean the ToolTip does not show up in the Unity editor when you hover over the field?

Comment (stbertou) [inline Assets/HoloToolkit/Utilities/Scripts/SphereBasedTagalong.cs]: Hey @NeerajW, yes indeed that's not correct, it should be fixed ! Sorry about that.

**Pull request merged**